perf/src/packet.rs

```diff
@@ -3,29 +3,87 @@ use crate::{cuda_runtime::PinnedVec, recycler::Recycler};
 use bincode::config::Options;
 use serde::Serialize;
 pub use solana_sdk::packet::{Meta, Packet, PACKET_DATA_SIZE};
-use std::net::SocketAddr;
+use std::{net::SocketAddr, time::Instant};
 
 pub const NUM_PACKETS: usize = 1024 * 8;
 
 pub const PACKETS_PER_BATCH: usize = 128;
 pub const NUM_RCVMMSGS: usize = 128;
 
+#[derive(Copy, Clone, Debug, Default)]
+pub struct PacketTimer {
+    incoming_start: Option<Instant>,
+    incoming_initial_end: Option<Instant>,
+    incoming_end: Option<Instant>,
+    outgoing_start: Option<Instant>,
+    num_coalesced: usize,
```

Comment:
maybe a comment on these, especially distinguishing `incoming_initial_end` and `incoming_end`

Reply:
removed incoming_initial_end